### PR TITLE
Added: New PasswordInput component

### PR DESCRIPTION
**Pull request by Miexil**
### What does this PR do?

Adds a new password-input component.
InputContainer can now handle error messages.

### Good PR checklist

- [x] Title makes sense
- [x] Is against the correct branch
- [x] Only addresses one issue
- [x] Properly assigned
- [x] Added/updated tests
- [x] Added/updated documentation
- [x] Migrated touched components to Glimmer Components
- [x] Properly labeled

### CODE REVIEW

File: addon/components/o-s-s/input-container.hbs [hunk 1]
```diff
@@ -1,17 +1,26 @@
-<div class="oss-input-container{{if (has-block "prefix") ' has-prefix'}}{{if (has-block "suffix") ' has-suffix'}}"
-     ...attributes>
-  {{#if (has-block "prefix")}}
-    <div class="prefix">{{yield to="prefix"}}</div>
-  {{/if}}
+<div class="fx-col">
+  <div class="oss-input-container {{if @errorMessage ' oss-input-errored'}}
```
Comment (phndiaye) [inline addon/components/o-s-s/input-container.hbs:2]: `oss-input-container--errored` ? :p

File: addon/components/o-s-s/password-input.stories.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+const DefaultUsageTemplate = (args) => ({
+  template: hbs`
+      <OSS::PasswordInput @value={{this.value}} />
+  `,
+  context: args
+});
+export const BasicUsage = DefaultUsageTemplate.bind({});
+BasicUsage.args = {
+  value: 'myPassword',
```
Comment (phndiaye) [inline addon/components/o-s-s/password-input.stories.js:38]: so strong !

Comment (Miexil) [reply addon/components/o-s-s/password-input.stories.js:38]: 😂

File: addon/components/o-s-s/input-container.hbs (hunk 1 above)
Comment (Miexil) [reply addon/components/o-s-s/input-container.hbs:2]: Can do !

**Review by JulienVannier66 — APPROVED** [review]
LGTM

**Review by phndiaye — APPROVED** [review]

**Pull request merged**